bundles/mapping/drawtools/event/DrawingEvent.js

```diff
@@ -11,11 +11,13 @@ Oskari.clazz.define('Oskari.mapping.drawtools.event.DrawingEvent',
  * @param {Object} geojson drawn shape (includes buffer as property?)
  * @param {Object} additional info like length of line or area of a polygon?
  */
-function(id, geojson, data, isFinished) {
+function(id, geojson, data, crs, isFinished, olObject) {
     this._id = id;
     this._geojson = geojson;
     this._data = data;
+    this._crs = crs;
     this._isFinished = isFinished;
+    this._olObject = olObject;
```

Comment:
Events should be serializable as JSON and shouldn't include any mapping lib specific values.